geopandas/tests/test_vectorized.py

```diff
@@ -333,6 +333,53 @@ def test_pickle():
     assert T.equals(T2).all()
 
 
+@pytest.mark.parametrize('predicate', [
+    'contains',
+    'covers',
+    'crosses',
+    'disjoint',
+    # 'equals',
```

Comment:
What is the reason that this is commented out (seems useful)? Because equals has no prepared version?